etc/hub.api.md

```diff
@@ -289,63 +327,49 @@ export class Client {
     delete(threadID: ThreadID, collectionName: string, IDs: string[]): Promise<void>;
     deleteCollection(threadID: ThreadID, name: string): Promise<void>;
     deleteDB(threadID: ThreadID): Promise<void>;
-    find<T = any>(threadID: ThreadID, collectionName: string, query: QueryJSON): Promise<InstanceList<T>>;
-    findByID<T = any>(threadID: ThreadID, collectionName: string, ID: string): Promise<Instance<T>>;
+    find<T = unknown>(threadID: ThreadID, collectionName: string, query: QueryJSON): Promise<T[]>;

```

Comment:
Lots of changes in here unrelated to my updates, not sure what's up with that.

Reply:
Huh, they must not have updated in my PR?

Reply:
All good though, these are correct.